### DISCUSSION TITLE
Version 7 causes compilation issues due to 'unexpected any', is this as intended?

### DISCUSSION THREAD

**Discussion by PieterBoeren** [nartc/mapper#377] [Q&A] [answered]
Hi,

Not sure if this is working as intended or not, but with version 6.* this code worked perfectly (profile):

```TypeScript
 mapper
    .createMap(SomeDto, SomeViewModel)
    .forMember(
      (destination) => destination.someValue,
      mapFrom((source) => new Date(source.someOtherValue))
    )
```

(Basically same as in the [documentation](https://automapperts.netlify.app/docs/getting-started/introduce-to-profile))

But in version 7, this all of a sudden gives me compilation errors like:

`TS7006: Parameter 'destination' implicitly has an 'any' type.`

It's fixable by supplying the correct types (`destination: SomeViewModel` and `source: SomeDto`), but is this expected? And if so, it should probably be added to the breaking changes I guess.

**Comment by nartc** [comment, marked as answer]
Have you changed the type import from automapper/types to automapper/core? It is a breaking change

**Reply by PieterBoeren** [reply]
Ah, my mistake :)

I got so many errors, that I didn't see the failing import. I did uninstall the types package, just didn't remove the imports...

**Reply by nartc** [reply]
I should have changed the documentation first before releasing 🤦‍♂️

**Reply by micalevisk** [reply]
every time I face a major upgrade, I go to releases page to read about the breaking changes :p

![image](https://user-images.githubusercontent.com/13461315/141482183-a41e6516-e793-4a47-9e03-02f4f1909437.png)

**Reply by PieterBoeren** [reply]
Me as well, just forgot to update my imports (TypeScript should stop emitting other errors if the imports are failing :)).

**Reply by nartc** [reply]
Updated docs site :D

**Reply by nartc** [reply]
@micalevisk Release now contains the CHANGELOG information as well.
![image](https://user-images.githubusercontent.com/25516557/141508810-4f9a7409-5f4d-48ec-b538-089b8e39448b.png)
